winter/http/request_body_resolver.py

```diff
@@ -20,7 +21,11 @@ def is_supported(self, argument: ComponentMethodArgument) -> bool:
 
     def resolve_argument(self, argument: ComponentMethodArgument, http_request: Request):
         fields = dataclasses.fields(argument.type_)
-        input_data = self._get_input_data(fields, http_request)
+        if isinstance(http_request.data, QueryDict):
+            input_data = self._get_input_data(fields, http_request)
```

Comment:
Maybe rename `_get_input_data` to `_convert_querydict_to_dict` or something?

Reply:
And maybe extract this part to another class, so that it would be more clear what the resolver does.